README.md

```diff
@@ -20,26 +20,52 @@ Or install it yourself as:
 
 ## Usage
 
-This API is provisional and is likely to change for 0.x versions.
+The API is provisional and is likely to change for versions < 1.0.0.
+
+To access the taxonomy, first request the content from the [publishing api](https://github.com/alphagov/publishing-api), then parse it to get a `LinkedEdition` object.
 
 ```ruby
 require 'plek'
 require 'gds_api/publishing_api'
 require 'govuk_taxonomy_helpers'
 
-content_store = GdsApi::ContentStore.new(Plek.new.find('content-store'))
-content_item = content_store.content_item('/education')
-taxonomy = GovukTaxonomyHelpers::Taxon.new(content_item)
+publishing_api = GdsApi::PublishingApi.new(Plek.new.find('publishing-api'))
+content_item = publishing_api.get_content("c75c541-403f-4cb1-9b34-4ddde816a80d")
+expanded_links = publishing_api.get_expanded_links("c75c541-403f-4cb1-9b34-4ddde816a80d")
+
+taxonomy = GovukTaxonomyHelpers.parse_publishing_api_response(
+  content_item: content_item,
+  expanded_links: expanded_links
+)
+```
+
+A `LinkedEdition` built from a taxon can access all *narrower term* taxons below it and all *broader term* taxons above it.
+
+A taxon may have many child taxons, but can only have one or zero parents.
 
-puts taxonomy.tree.map(&:name) # All taxons
-puts taxonomy.descendants(&:name) # All descendant taxons
-puts taxonomy.children.map(&:name) # All direct children
+```ruby
+puts taxonomy.tree             # All taxons in this branch of the taxonomy
```

Comment:
Perhaps pasting the outputs here would be good. Alternatively, referring to the method docs.